downstream-changes/update.sh

```diff
@@ -0,0 +1,81 @@
+#/bin/bash
+set -eux -o pipefail
+
+UPSTREAM_CLONE_URL=${UPSTREAM_CLONE_URL:-https://github.com/istio/istio.git}
+DOWNSTREAM_CLONE_URL=${DOWNSTREAM_CLONE_URL:-https://github.com/openshift-service-mesh/istio.git}
+REPO=$(basename "${UPSTREAM_CLONE_URL}" .git)
+BRANCHES=${BRANCHES:-master release-1.24}
```

Comment:
Do we want to include `release-1.25` as well? Not sure since it's not included in the downstream operator as of `3.0.0`.

Reply:
[it doesn't exist yet](https://github.com/openshift-service-mesh/istio/tree/release-1.25) :) we can add it here once we create it